apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,38 @@
+
+#include "TutorialServo.h"
+
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1,
+                             float maxPulsewidthInMs = 2) {
+  m_servoPwmOut = servoPin;
+  m_servoPwmOut.period_ms(20);
+  m_servoRangeInDegrees = servoRangeInDegrees;
+  m_minPulsewidthInMs   = minPulsewidthInMs;
+  m_maxPulsewidthInMs   = maxPulsewidthInMs;
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+  float pulse = m_minPulsewidthInMs + (degrees / m_servoRangeInDegrees);

```

Comment:
What if `m_maxPulsewidthInMs` was 3ms instead of 2ms, would this still be the correct mapping?